templates/graphdb-node.yaml

```diff
@@ -120,6 +120,11 @@ spec:
             - name: graphdb-node-storage
             {{- end }}
               mountPath: /opt/graphdb/home
+            {{- if $.Values.graphdb.node.license }}
+            - name: graphdb-license
+              mountPath: /opt/graphdb/home/conf/graphdb.license
+              subPath: graphdb.license
+            {{- end }}
```

Comment:
It won't because of the `subPath` which will take graphdb.license from the secret and place it as a file.

Reply:
I see. Well, I'm gonna add a configuration for the filename just in case someone wants to use different naming (although `values.yaml` states that we expect `graphdb.license`).

Reply:
Posted https://ontotext.atlassian.net/browse/GDB-8443